articles/sentinel/ci-cd.md

```diff
@@ -120,7 +120,9 @@ Each connection can support multiple types of custom content, including analytic
 
 After the connection is created, a new workflow or pipeline is generated in your repository, and the content stored in your repository is deployed to your Azure Sentinel workspace.
 
-The deployment time may vary depending on the amount of content that you're deploying. View the deployment status:
+The deployment time may vary depending on the amount of content that you're deploying. 
```

Comment:
Hi @nayef-yassin - thanks! I think it wasn't actually, but we can make it into a heading if you prefer.
In this context, I think better an H3 though than an H2.
Can you modify to have 3 hashes, and no colon?

### View the deployment status

Reply:
Sounds good thanks!